### DISCUSSION TITLE
Caputo型分数阶微分方程的损失函数代码问题

### DISCUSSION THREAD

**Discussion by qiuzlong** [lululxvi/deepxde#766] [Q&A]
您好
![image](https://user-images.githubusercontent.com/83164134/175767185-0476585c-9fb1-4870-990a-96d0de7a67cf.png)
请问这样的方程的损失函数如何定义，代码怎么写

**Comment by lululxvi** [comment]
See https://doi.org/10.1137/18M1229845
